.importanizerc

```diff
@@ -9,6 +9,9 @@
         {"type": "packages", "packages": ["sqlalchemy"]},
         {"type": "packages", "packages": ["django"]},
         {"type": "packages", "packages": ["django_sorcery"]},
+        {"type": "packages", "packages": ["rest_framework"]},
```

Comment:
really need to ship the new importanize version when I fix all the tests which allows setup.cfg configs...

Reply:
yes pls